plugins/aea-crypto-cosmos/aea_crypto_cosmos/cosmos.py

```diff
@@ -62,6 +62,15 @@
 DEFAULT_CHAIN_ID = "cosmoshub-3"
 _BYTECODE = "wasm_byte_code"
 DEFAULT_CLI_COMMAND = "wasmcli"
+DEFAULT_COSMWASM_VERSIONS = "<0.10"
+MESSAGE_FORMAT_BY_VERSION = {
+    "store": {"<0.10": "wasm/store-code", ">=0.10": "wasm/MsgStoreCode"},
```

Comment:
This might be a little bit tricky because FetchD reports different versions than CosmWasm. For example, FetchD v0.5.0 is using CosmWasm 0.10.x. We just have to keep it in mind.

Reply:
Yes, this is purely on the Python side though, so no need for them to match strictly.

Reply:
Adding full version checks is a bit overkill here imo. Means another dependency to parse version etc.